mne_bids/utils.py

```diff
@@ -503,6 +503,29 @@ def _get_brainvision_paths(vhdr_path):
     return (eeg_file_path, vmrk_file_path)
 
 
+def copyfile_ctf(src, dest):
+    """Copy and rename CTF files to a new location.
+
+    Parameters
+    ----------
+    src : str
+        path to the source raw .ds folder
+    dest : str
+        path to the destination of the new bids folder.
+    """
+    sh.copytree(src, dest)
+    # list of file types to rename
+    file_types = ('.acq', '.eeg', '.hc', '.hist', '.infods', '.infods.bak',
```

Comment:
is there an easy way to link to a relevant part in the documentation of the CTF data format? Any online ressource? Otherwise referring to a section in some PDF that CTF customers/users have access to?

Otherwise, these filenames and the reasoning come a bit out of the blue

Reply:
I'll check if I find a PDF online describing this format.

Reply:
> Surprinsingly, the CTF website doesn't have a PDF guide to the CTF MEG and format.

okay, that's sad ...

>  Do you think a link to that PDF is good?

not sure ... it's also from 2006 ^_^ so either outdated or really stable and no one ever added a feature or found a bug.

I think we are good to go with this PR.